integreat_cms/static/src/js/pages/page-api-token.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Generate and (un)set the API token.
+ */
+
+// Hack to get around the fact that randomUUID is not yet a part of the default TypeScript libs
+declare global {
+  interface Crypto {
+    randomUUID: () => string;
+  }
+}
```

Comment:
```suggestion
```
I'd prefer to just do an `npm update`, because newer versions of typescript already support this.